instrumentation/github.com/syndtr/goleveldb/leveldb/splunkleveldb/README.md

```diff
@@ -0,0 +1,16 @@
+# Splunk instrumentation for `github.com/syndtr/goleveldb/leveldb`
+
+This package provides OpenTelemetry instrumentation for the
+[github.com/syndtr/goleveldb/leveldb](https://github.com/syndtr/goleveldb/leveldb)
+package.
+
+## Compatibility
+
+- Go: 1.15+
+- github.com/syndtr/goleveldb/leveldb: 1.0.0+
```

Comment:
It is the first time I see such a section in this repository 😄 
Is it intended?

Reply:
It was intentional, but ultimately it is not needed. I'll remove it to remain consistent and progress the PR.